src/hooks/createUseApi.js

```diff
@@ -130,7 +130,13 @@ function createUseApi(apiTypes: ApiTypeMap) {
     }, [currentEntityState])
 
     useFetchOnInitialMount(method, lazy, entityState, promiseProp)
-    useReFetchOnRefresh(refresh, method, lazy, promiseProp)
+    useReFetchOnRefresh(
+      refresh,
+      method,
+      lazy,
+      promiseProp,
+      requestState?.refresh
```

Comment:
Should have said this before. Can you move the `requestState?.fresh` in front or directly after the `refresh` parameter? My very personal opinion is that similar parameters should be close to another.

Reply:
done